documentation/AuthGrantFinder.md

```diff
@@ -1,6 +1,6 @@
 ## Authorization Grant Finder
 
-The authorization grant finder provides API that allows users to use access tokens to retrieve their corresponding authorization information. The access tokens can be requested via [GPII OAuth2 authorization code grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Authorization_Code_Grant) or [GPII OAuth2 client credentials grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Client_Credentials_Grant).
+The authorization grant finder provides API that allows users to use access tokens to retrieve their corresponding authorization information. The access tokens can be requested via [Resource Owner GPII Token Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Resource_Owner_GPII_Token_Grant).
 
 #### getGrantForAccessToken(accessToken)
```

Comment:
I think we can remove the sentence "The object can have a different data structure when the access token is requested via different OAuth2 grant methods for different client types." as we only have one grant type now.

And we can change "An example of the returned object when the access token is granted to GPII app installations via resource owner GPII token grant:" to "An example of the returned object:".